libvcx/src/v3/handlers/connection/connection.rs

```diff
@@ -169,6 +169,13 @@ impl Connection {
     pub fn send_message(&self, message: &A2AMessage) -> VcxResult<()> {
         trace!("Connection::send_message >>> message: {:?}", message);
 
+        match self.state_object() {
```

Comment:
Won't this be producing warn messages during within the connection protocol itself?

Reply:
Send message is not used during the connection protocol.